anndata/_core/anndata.py

```diff
@@ -91,11 +91,13 @@ def _check_2d_shape(X):
 
 
 def _gen_dataframe(anno, length, index_names):
-    if isinstance(anno, pd.DataFrame):
+    if isinstance(anno, pd.DataFrame) or isinstance(anno, pd.Series):
```

Comment:
If you want to keep this instead of going for the idea in the other comment, do
```suggestion
    if isinstance(anno, (pd.DataFrame, pd.Series)):
```